src/ui/preferences.ui

```diff
@@ -176,190 +278,222 @@ however, the article from the topmost dictionary is shown.</string>
-            <string>Turn the UI to dark.</string>
-           </property>
-           <property name="text">
-            <string>Dark Mode</string>
-           </property>
-          </widget>
-         </item>
-         <item>
-          <widget class="QCheckBox" name="darkReaderMode">
-           <property name="toolTip">
-            <string>Turn the article display style to dark.</string>
-           </property>
-           <property name="text">
-            <string>Dark Reader Mode</string>
-           </property>
-          </widget>
-         </item>
-        </layout>
-       </item>
-       <item row="3" column="0">
-        <widget class="QCheckBox" name="selectBySingleClick">
-         <property name="toolTip">
-          <string>Turn this option on if you want to select words by single mouse click</string>
-         </property>
-         <property name="text">
-          <string>Select word by single click</string>
-         </property>
-        </widget>
-       </item>
-       <item row="6" column="0">
-        <widget class="QComboBox" name="interfaceLanguage">
-         <property name="sizeAdjustPolicy">
-          <enum>QComboBox::AdjustToContents</enum>
-         </property>
-        </widget>
-       </item>
-       <item row="8" column="0">
-        <widget class="QFontComboBox" name="fontFamilies">
-         <property name="sizePolicy">
-          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
-           <horstretch>0</horstretch>
-           <verstretch>0</verstretch>
-          </sizepolicy>
-         </property>
-         <property name="toolTip">
-          <string>set the fallback font family for dictionary</string>
-         </property>
-        </widget>
-       </item>
-       <item row="2" column="0">
-        <widget class="QCheckBox" name="doubleClickTranslates">
-         <property name="text">
-          <string>Double-click translates the word clicked</string>
-         </property>
-        </widget>
-       </item>
-       <item row="0" column="0" colspan="2">
-        <widget class="QGroupBox" name="groupBox">
+       <item row="9" column="1">
+        <widget class="QGroupBox" name="groupBox_10">
          <property name="title">
-          <string>Tabbed browsing</string>
+          <string>Customize Fonts</string>
          </property>
-         <layout class="QGridLayout" name="gridLayout_5">
-          <item row="0" column="0">
-           <widget class="QCheckBox" name="newTabsOpenInBackground">
-            <property name="toolTip">
-             <string>Normally, opening a new tab switches to it immediately.
-With this on however, new tabs will be opened without
-switching to them.</string>
+         <layout class="QVBoxLayout" name="verticalLayout_17">
+          <item>
+           <widget class="QLabel" name="label_7">
+            <property name="text">
+             <string>These fonts will be applied if the dictionary's specified fonts have not been found.</string>
```

Comment:
How about changing the "Customize fonts" to "Fallback fonts" in which the word "Fallback" includes every meaning of this sentence?

`dictionary's specified fonts` sounds weird.

`These fonts will be applied when the fonts specified by a dictionary are not found.` feels more natural.

If we use fallback fonts as the title, we can move this further explanation sentence to the tooltip, maybe?

![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/d20ca802-55d4-4a39-9839-5fbb4686fd1c)